src/containers/components/buttonGroup/buttonGroup.container.tsx

```diff
@@ -1,24 +1,49 @@
 import React from 'react';
 import { NavigationScreenProps } from 'react-navigation';
-import { ButtonGroupProps } from '@kitten/ui';
-import { ButtonGroupShowcase } from './buttonGroupShowcase.component';
 import { Showcase } from '../common/showcase.component';
-import { buttonGroupShowcase } from './type';
+import { ShowcaseSection } from '../common/showcaseSection.component';
+import {
+  FilledButtonGroup,
+  GiantButtonGroup,
+  LargeButtonGroup,
+  MediumButtonGroup,
+  OutlineButtonGroup,
+  SmallButtonGroup,
+  TinyButtonGroup,
+} from './showcase';
+import { ShowcaseItem } from '@src/containers/components/common/showcaseItem.component';
 
 export class ButtonGroupContainer extends React.Component<NavigationScreenProps> {
 
-  private renderItem = (props: ButtonGroupProps): React.ReactElement<ButtonGroupProps> => {
-    return (
-      <ButtonGroupShowcase {...props}/>
-    );
-  };
-
   public render(): React.ReactNode {
     return (
-      <Showcase
-        showcase={buttonGroupShowcase}
-        renderItem={this.renderItem}
-      />
+      <Showcase>
+        <ShowcaseSection title='Appearance'>
+          <ShowcaseItem title='Filled'>
+            <FilledButtonGroup/>
```

Comment:
Don't see any sense in such example of wrapping component with useless wrapper

Reply:
Just to support the idea of creating all examples. And yes: now it can be default, but later it could not